frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue

```diff
@@ -20,19 +22,19 @@ const selection = ref<string>('')
 watch(selection, (value) => { network.value = Number(value) as ChainIDs })
 
 const buttonList = ValidatorDashboardNetworkList.map((chainId) => {
+  // TODO: simply set `false` for everything once dashboards can be created for all the networks in `ValidatorDashboardNetworkList`
+  const isDisabled = !useRuntimeConfig().public.showInDevelopment && chainId !== currentNetwork.value
   return {
     value: String(chainId),
     text: ChainInfo[chainId].family as string,
-    subText: (ChainInfo[chainId].name !== ChainInfo[chainId].family as string) ? ChainInfo[chainId].name : ChainInfo[chainId].description,
-    disabled: !useRuntimeConfig().public.showInDevelopment && chainId !== currentNetwork.value, // TODO: simply set `false` for everything once dashboards can be created for all the networks in `ValidatorDashboardNetworkList`
+    subText: isDisabled ? $t('dashboard.creation.coming_soon') : ChainInfo[chainId].name,
```

Comment:
**Bug:** Using `isDisabled` like this is good but instead of `: ChainInfo[chainId].name` you will have to use everything that was used for `subText` before, namely `(ChainInfo[chainId].name !== ChainInfo[chainId].family as string) ? ChainInfo[chainId].name : ChainInfo[chainId].description`.

Because without that, the Ethereum button looks like this
![image](https://github.com/gobitfly/beaconchain/assets/114066135/c71fe627-a034-4ca2-bc45-ad187e50a75b)
while it should look like this
![image](https://github.com/gobitfly/beaconchain/assets/114066135/02593dc6-fb71-436b-b5b4-9bc5f8dd2077)

Reply:
thx done